### PR TITLE
feat: automated validation of tokens decimals

### PR DESCRIPTION
**Pull request by smol-ninja**
Closes https://github.com/sablier-labs/community-token-list/issues/67.

### Changelog

- [x] Adds a Github workflow to validate token decimals
- [x] Adds bash script to manually run checks on all tokens listed
- [x] Removes redundant/invalid tokens from the list

### CODE REVIEW

**Comment by razgraf** [comment]
Nice nice nice @smol-ninja!

CC: @gavriliumircea and @maxdesalle, whenever we add a new chain, let's also make sure we add the RPCs and necessary configurations to this GA.

**Review by razgraf — APPROVED** [review]

**Comment by smol-ninja** [comment]
Things to do when a new chain is added:
1. Add `<CHAIN_NAME>_MAINNET_RPC_URL` to `.env.example`.
2. Add `<CHAIN_NAME>_MAINNET_RPC_URL` to `tokenlist.yml` workflow.

If it is available on Infura, add infura endpoint, if its not available on Infura, any relevant RPC from chainlist can be added.

The tokenlist CI will run on every PR and on every commit to `main`. It will also run once a week.

**Comment by smol-ninja** [comment]
This is how a successful CI run would look like 👇

Screenshot 2024-12-02 at 20 40 11

File: .env.example [hunk 1]
```diff
@@ -0,0 +1,27 @@
+export API_KEY_INFURA="YOUR_RPC_API_KEY"
+
+# Mainnets
+export ARBITRUM_MAINNET_RPC_URL="https://arbitrum-mainnet.infura.io/v3/$API_KEY_INFURA"
+export AVALANCHE_MAINNET_RPC_URL="https://avalanche-mainnet.infura.io/v3/$API_KEY_INFURA"
+export BASE_MAINNET_RPC_URL="https://base-mainnet.infura.io/v3/$API_KEY_INFURA"
+export BLAST_MAINNET_RPC_URL="https://blast-mainnet.infura.io/v3/$API_KEY_INFURA"
+export BSC_MAINNET_RPC_URL="https://bsc-mainnet.infura.io/v3/$API_KEY_INFURA"
+export ETHEREUM_MAINNET_RPC_URL="https://mainnet.infura.io/v3/$API_KEY_INFURA"
+export GNOSIS_MAINNET_RPC_URL="YOUR_GNOSIS_MAINNET_RPC_URL"
```
Comment (PaulRBerg) [inline .env.example:10]: Do we really need environment variables for this?

Can't we use public RPC URLs that can be found on Chainlist?

Comment (smol-ninja) [reply .env.example:10]: Good point and we can. I will do that in that case.

Comment (PaulRBerg) [reply .env.example:10]: Great, thanks. That should lower the maintenance cost (as we need fewer environment variables).

Comment (smol-ninja) [reply .env.example:10]: https://github.com/sablier-labs/community-token-list/commit/e321158a1f76c7c0fbdeb866c6c0dc036378e249

**Pull request merged**